rct229/rulesets/ashrae9012019/section5/section5rule23.py

```diff
@@ -110,3 +111,9 @@ def rule_check(self, context, calc_vals=None, data=None):
                         calc_vals["subsurface_p_manual_shade"]
                         == calc_vals["subsurface_b_manual_shade"]
                     )
+
+                def is_tolerance_fail(self, context, calc_vals=None, data=None):
```

Comment:
This rule does not need to have a tolerance failure because the data type is boolean.
`calc_vals["subsurface_p_manual_shade"]` is a `True` or `False`.
The `is_tolerance_fail` only checks on numerical values.